runtime/tests/runtime_integration_tests/test_get_idev_csr.rs

```diff
@@ -25,20 +26,23 @@ fn test_get_csr() {
         chksum: caliptra_common::checksum::calc_checksum(u32::from(CommandId::GET_IDEV_CSR), &[]),
     };
 
-    let response = model
-        .mailbox_execute(CommandId::GET_IDEV_CSR.into(), payload.as_bytes())
-        .unwrap()
-        .unwrap();
+    let result = model.mailbox_execute(CommandId::GET_IDEV_CSR.into(), payload.as_bytes());
+
+    match get_ci_rom_version() {
```

Comment:
You probably won't technically need this version check here since this will be supported starting the in the 2.0 ROM

Reply:
Ack.